modular_zubbers/code/datums/quirks/positive_quirks/venemous_bite/venomous_bite.dm

```diff
@@ -0,0 +1,124 @@
+#define VENOMOUS_BITE_DAMAGE 5
+#define VENOMOUS_BITE_WOUND_CHANCE 20
+#define VENOMOUS_BITE_WOUND_BONUS 40
+
+/datum/action/cooldown/mob_cooldown/venomous_bite
+	name = "Inject Venom"
+	desc = "Sink your fangs into another, and fill them with your venom. Not guaranteed to work against armor."
+
+	button_icon = 'modular_zubbers/icons/mob/actions/bloodsucker.dmi'
+	button_icon_state = "power_feed"
+
+	ranged_mousepointer = 'icons/effects/mouse_pointers/supplypod_pickturf.dmi'
+
+	/// The reagent we will inject.
+	var/datum/reagent/reagent_typepath
+	/// How much of [reagent_typepath] we will inject.
+	var/to_inject = 0
+
+/datum/action/cooldown/mob_cooldown/venomous_bite/New(Target, original, datum/reagent/our_reagent, quantity_override = null)
+	. = ..()
+
+	set_reagent(our_reagent, quantity_override)
+
+/datum/action/cooldown/mob_cooldown/venomous_bite/proc/set_reagent(datum/reagent/new_reagent, quantity_override = null, cooldown_override = null)
+	reagent_typepath = new_reagent
+
+	var/list/reagent_spec = GLOB.venomous_bite_choice_specs[new_reagent]
+	if (isnum(quantity_override))
+		to_inject = quantity_override
+	else
+		to_inject = reagent_spec[1]
+
+	if (isnum(cooldown_override))
+		cooldown_time = cooldown_override
+	else
+		cooldown_time = reagent_spec[2]
+
+/datum/action/cooldown/mob_cooldown/venomous_bite/set_click_ability(mob/on_who)
+	. = ..()
+	if (!.)
+		return
+
+	owner.visible_message("[owner] bares [owner.p_their()] fangs...", span_warning("You bare your fangs..."))
+
+/datum/action/cooldown/mob_cooldown/venomous_bite/Activate(atom/target_atom)
+	if (!isliving(target_atom))
+		return FALSE
+	if (iscarbon(owner))
+		var/mob/living/carbon/carbon_holder = owner
+		if (carbon_holder.is_mouth_covered())
+			owner.balloon_alert("mouth covered!")
+			return FALSE
```

Comment:
Add a check that stops you from biting yourself.

Reply:
may i ask why?

Reply:
It's kind of weird to bite yourself in the chest or something. I can understand if it's in your arm, but truth be told I would rather just make it so you can't in the first place so we don't have to worry about people targetting themselves with any of the chems added